### PR TITLE
Add docs about using simplewallet without dockers

### PR DESCRIPTION
**Pull request by askmish**
Signed-off-by: Ashish Kumar Mishra <ashish.k.mishra@intel.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,6 +112,29 @@ docker-compose -f simplewallet-build-tp-<your_prog_language>.yaml up --build
 where,
  <your_prog_language> should be replaced with either `cxx`, `java`, or `py`
 
+# Building and running on OS(without dockers)
+To run sawtooth-simplewallet without dockers, we'll have to use a Ubuntu 16.04 OS isnatllataion and compile simplewallet from sources. Below is a sample procedure for python TP/client:
```
Comment (danintel) [inline README.md]: s/isnatllataion/installation/

File: README.md [hunk 2]
```diff
@@ -112,6 +112,29 @@ docker-compose -f simplewallet-build-tp-<your_prog_language>.yaml up --build
...
+
+1. Install sawtooth on Ubuntu 16.04 LTS x64 machine. Refer sawtooth app developer's guide [here](https://sawtooth.hyperledger.org/docs/core/releases/latest/app_developers_guide/ubuntu.html)
+2. Clone sawtooth-simplewallet repo
```
Comment (danintel) [inline README.md]: s:Clone sawtooth-simplewallet repo:git clone https://github.com/askmish/sawtooth-simplewallet/

File: README.md [hunk 3]
```diff
@@ -112,6 +112,29 @@ docker-compose -f simplewallet-build-tp-<your_prog_language>.yaml up --build
...
+3. `cd sawtooth-simplewallet`
+4. Setup the simplewallet-tp.
+ - Follow the [`pyprocessor/Dockerfile`](https://github.com/askmish/sawtooth-simplewallet/blob/master/pyprocessor/Dockerfile)
+ - Install all the dependencies in the [first `RUN` line](https://github.com/askmish/sawtooth-simplewallet/blob/master/pyprocessor/Dockerfile#L18) in the Dockerfile
+ - Add the `pyprocessor` directory to your `PATH` environment, as shown below:
+
+`export PATH = $PATH:<absolute-path-to-your-cloned-sawtooth-simplewallet-dir>/pyprocessor"`
+ - Run the simplewallet-tp by following command:
+
+   `simplewallet-tp`
```
Comment (danintel) [inline README.md]: I would remove the step for adding the `pyprocessor` directory to `PATH` (lines 124-129) and replace with this instruction instead:

> Start the simplewallet transaction processor with `./pyprocessor/simplewallet-tp`

File: README.md [hunk 4]
```diff
@@ -112,6 +112,29 @@ docker-compose -f simplewallet-build-tp-<your_prog_language>.yaml up --build
...
+5. Setup the client. Open a new console.
+ - Follow the [`pyclient/Dockerfile`](https://github.com/askmish/sawtooth-simplewallet/blob/master/pyclient/Dockerfile)
+ - Install all the dependencies in the [first `RUN` line](https://github.com/askmish/sawtooth-simplewallet/blob/master/pyclient/Dockerfile#L20) in the pyclient/Dockerfile
+ - Add the `pyclient` directory to your `PATH` environment, as shown below:
+
+`export PATH = $PATH:<absolute-path-to-your-cloned-sawtooth-simplewallet-dir>/pyclient"`
+ - Run the client by calling `simplewallet` command. Refer Usage section above.
+
```
Comment (danintel) [inline README.md]: I would remove the step for adding the `pyclient` directory to `PATH` (lines 133-136) and replace with this instruction instead:

> Start the simplewallet client with `./pyclient/simplewallet`

**Review by danintel — CHANGES_REQUESTED** [review]
I made some suggestions to the instructions for starting the processor and client.

This is a good idea--I've seen requests for non-Docker instructions a few times on Rocketchat.

File: README.md [hunk 5]
```diff
@@ -112,6 +112,25 @@ docker-compose -f simplewallet-build-tp-<your_prog_language>.yaml up --build
 where,
  <your_prog_language> should be replaced with either `cxx`, `java`, or `py`
 
+# Building and running on OS(without dockers)
+To run sawtooth-simplewallet without dockers, we'll have to use a Ubuntu 16.04 OS installation and compile simplewallet from sources. Below is a sample procedure for python TP/client:
+
+1. Install sawtooth on Ubuntu 16.04 LTS x64 machine. Refer sawtooth app developer's guide [here](https://sawtooth.hyperledger.org/docs/core/releases/latest/app_developers_guide/ubuntu.html)
+2. git clone https://github.com/askmish/sawtooth-simplewallet.git
+3. `cd sawtooth-simplewallet`
```
Comment (danintel) [inline README.md]: I just tried it and two files need to be modified.  So here are some more instructions:

Add a step:

3A Modify the transaction processor and client as follows:
- `git branch nodocker; git checkout nodocker`
- Edit file `pyclient/wallet/simplewallet_cli.py` and change `rest-api:8008` to `localhost:8008`
- Edit file `pyprocessor/processor/simplewallet_tp.py` and change `validator:4004` to `localhost:4004`

File: README.md [hunk 6]
```diff
@@ -112,6 +112,25 @@ docker-compose -f simplewallet-build-tp-<your_prog_language>.yaml up --build
...
+4. Setup the simplewallet-tp.
+ - Follow the [`pyprocessor/Dockerfile`](https://github.com/askmish/sawtooth-simplewallet/blob/master/pyprocessor/Dockerfile)
+ - Install all the dependencies in the [first `RUN` line](https://github.com/askmish/sawtooth-simplewallet/blob/master/pyprocessor/Dockerfile#L18) in the Dockerfile
```
Comment (danintel) [inline README.md]: - Start the validator, REST API, and Settings TP in separate consoles:
`sudo -u sawtooth sawtooth-validator -vv`
`sudo -u sawtooth sawtooth-rest-api -vvv`
`sudo -u sawtooth settings-tp -vv`

**Review by danintel — CHANGES_REQUESTED** [review]
Added instructions to 
* change the `validator` and `rest-api` URLs to `localhost`
* start the validator, rest-api, and settings TP

File: README.md (hunk 5 above)
Comment (askmish) [reply README.md]: Ah, I forgot to add these info. Thanks.

File: README.md [hunk 7]
```diff
@@ -112,6 +112,25 @@ docker-compose -f simplewallet-build-tp-<your_prog_language>.yaml up --build
 where,
  <your_prog_language> should be replaced with either `cxx`, `java`, or `py`
 
+# Building and running on OS(without dockers)
+To run sawtooth-simplewallet without dockers, we'll have to use a Ubuntu 16.04 OS installation and compile simplewallet from sources. Below is a sample procedure for python TP/client:
+
+1. Install sawtooth on Ubuntu 16.04 LTS x64 machine. Refer sawtooth app developer's guide [here](https://sawtooth.hyperledger.org/docs/core/releases/latest/app_developers_guide/ubuntu.html)
```
Comment (danintel) [inline README.md]: Add the following:

> and create the Genesis Block

**Pull request merged**